Translations/en.js

```diff
@@ -161,6 +161,28 @@ _global.translation['en'] = {
+    ,"1_3_5_H98.InvalidAutocomplete_Multiline" : 'Invalid autocomplete value: {{x}}. Element does not belong to Multiline control group. See <a href=\"https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute\">https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute</a>'
+    ,"1_3_5_H98.InvalidAutocomplete_Password" : 'Invalid autocomplete value: {{x}}. Element does not belong to Password control group. See <a href=\"https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute\">https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute</a>'
+    ,"1_3_5_H98.InvalidAutocomplete_Url" : 'Invalid autocomplete value: {{x}}. Element does not belong to Url control group. See <a href=\"https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute\">https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute</a>'
+    ,"1_3_5_H98.InvalidAutocomplete_Telephone" : 'Invalid autocomplete value: {{x}}. Element does not belong to Telephone control group. See <a href=\"https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute\">https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute</a>'
+    ,"1_3_5_H98.InvalidAutocomplete_Numeric" : 'Invalid autocomplete value: {{x}}. Element does not belong to Numeric control group. See <a href=\"https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute\">https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute</a>'
+    ,"1_3_5_H98.InvalidAutocomplete_Month" : 'Invalid autocomplete value: {{x}}. Element does not belong to Month control group. See <a href=\"https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute\">https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute</a>'
+    ,"1_3_5_H98.InvalidAutocomplete_Date" : 'Invalid autocomplete value: {{x}}. Element does not belong to Date control group. See <a href=\"https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute\">https://www.w3.org/TR/html52/sec-forms.html#autofilling-form-controls-the-autocomplete-attribute</a>'
+    ,"1_3_5_H98.Purpose" : 'Check that the input field serves a purpose identified in the <a href="https://www.w3.org/TR/WCAG21/#input-purposes" target="_blank">Input Purposes for User Interface Components</a> section; and that the content is implemented using technologies with support for identifying the expected meaning for form input data.'
```

Comment:
I think it may be better to leave the additional markup out of the string translations. The Auditor UI can't display it correctly (this one gets cut off in the summary, and isn't styled by the Auditor CSS). There are links displayed by the auditor to relevant documentation that the user can follow up with for more detail.

Reply:
Will do